docs/api/address/get-address-assets.example.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This file has 4 space tabs, everywhere else is 2. `prettier --write docs/**/*.json`

Reply:
thanks, fixed!